app/models/merit/badge_rules.rb

```diff
@@ -0,0 +1,49 @@
+# Be sure to restart your server when you modify this file.
+#
+# +grant_on+ accepts:
+# * Nothing (always grants)
+# * A block which evaluates to boolean (recieves the object as parameter)
+# * A block with a hash composed of methods to run on the target object with
+#   expected values (+votes: 5+ for instance).
+#
+# +grant_on+ can have a +:to+ method name, which called over the target object
+# should retrieve the object to badge (could be +:user+, +:self+, +:follower+,
+# etc). If it's not defined merit will apply the badge to the user who
+# triggered the action (:action_user by default). If it's :itself, it badges
+# the created object (new user for instance).
+#
+# The :temporary option indicates that if the condition doesn't hold but the
+# badge is granted, then it's removed. It's false by default (badges are kept
+# forever).
+
+module Merit
+  class BadgeRules
+    include Merit::BadgeRulesMethods
+
+    def initialize
+      # If it creates user, grant badge
+      # Should be "current_user" after registration for badge to be granted.
+      # Find badge by badge_id, badge_id takes presidence over badge
+      # grant_on 'users#create', badge_id: 7, badge: 'just-registered', to: :itself
```

Comment:
Can we rip out the comments from this?  I know it's just generated but the merit docs are pretty good.  I don't think we need all this documentation in the codebase.